src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -679,5 +706,14 @@ private bool TrySetNestedResourceInternal(string name, object deltaNestedResourc
 
             return true;
         }
+
+        /// <summary>
+        /// GetEdmType method in herited from IEdmObject , not implemented
+        /// </summary>
+        /// <returns>Returns the EdmType of the Type</returns>
+        Microsoft.OData.Edm.IEdmTypeReference IEdmObject.GetEdmType()
```

Comment:
>GetEdmType [](start = 57, length = 10)

Is this a problem in any scenarios? Do we need a way to set an IEdmTypeReference (or model) on the Delta<TStructuralType> object?

Reply:
This is now supported (although may be null).  Please update the summary and call out in Returns that it may be null if not specified.

---
In reply to: [551592880](https://github.com/OData/WebApi/pull/2275#discussion_r551592880) [](ancestors = 551592880)